yt_dlp/extractor/vimeo.py

```diff
@@ -212,6 +212,14 @@ def _parse_config(self, config, video_id):
         owner = video_data.get('owner') or {}
         video_uploader_url = owner.get('url')
 
+        audio_quality_map = ('low', 'medium', 'high')
```

Comment:
Don't call this `_map`
Consider `ordered_audio_qualities`
Or change it to (i prefer the one below)
```
{
'low': 1,
'medium': 2,
'high': 3

}
```

Reply:
Yeah a dict was what I originally had. But when I looked around I found yt-dlp's source code "style" seems to prefer brevity so I changed to a tuple to be shorter. I can change it back to a dict.